src/index.js

```diff
@@ -127,6 +127,15 @@ app.setAppUserModelId('com.marshallofsound.gpmdp.core');
 
     // and load the index.html of the app.
     mainWindow.loadURL(`file://${__dirname}/public_html/index.html`);
+
+    // show the main window.
+    mainWindow.show();
```

Comment:
We don't need to show the window here, only to minimize it in the next conditional. We can safely put this in an else below since you've defaulted startMinimized to false. This will keep the application from "flashing" which I'm sure will be a future issue if this is implemented.